### ISSUE TITLE
`sparse_rec_fista` fails when using `pywt` wavelets

### ISSUE THREAD

**Issue by zaccharieramzi** [CEA-COSMIC/pysap-mri#3]
When trying to use `sparse_rec_fista` with `pywt` it fails with:

```python

~/workspace/pysap/pysap/numerics/reconstruct.py in sparse_rec_fista(gradient_op, linear_op, prox_op, cost_op, mu, nb_scales, lambda_init, max_nb_of_iter, atol, metric_call_period, metrics, verbose)
     85     # Check inputs
     86     start = time.clock()
---> 87     if not linear_op.transform.__is_decimated__:
     88         warnings.warn("Undecimated wavelets shouldn't be used with FISTA: "
     89                       "non optimal solution.")

AttributeError: 'Db4' object has no attribute '__is_decimated__'
```

This is because of the [lines where we check whether the wavelet is decimated](https://github.com/CEA-COSMIC/pysap/blob/master/pysap/numerics/reconstruct.py#L87-L89). We need to adapt these lines to not fail in the `pywt` case.

**Comment by philouc** [comment]
I agree with you. Go ahead and get rid of this check for the moment.

**Comment by zaccharieramzi** [comment]
Closed by #17
